runtime/common/src/lib.rs

```diff
@@ -822,6 +826,7 @@ macro_rules! impl_config_traits {
             type ProposalBondMaximum = ProposalBondMaximum;
             type RejectOrigin = EnsureRootOrTwoThirdsCouncil;
             type SpendFunds = Bounties;
+            type SpendOrigin = NeverEnsureOrigin<Balance>;
```

Comment:
I agree to take this defensive approach first :+1: . No origin can directly spend funds from the treasury using this configuration. Maybe we replace that by a dedicated treasury council in the future.